simplifier/operations/interface.py

```diff
@@ -123,4 +125,10 @@ def simplify(self, keep_form: bool = False) -> Optional[WorldObject]:
         """
         self._promote_subexpression()
         self._fold_constants()
-        return super().simplify(keep_form)
+        op = super().simplify(keep_form)
```

Comment:
```suggestion
        simplified_operation = super().simplify(keep_form)
```
+ change the name in the following. I prefer names that are more descriptive :)